modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -55,7 +54,12 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
 
           data = power_of_attorney.form_data
 
-          if data.dig('signatures', 'veteran').present? && data.dig('signatures', 'representative').present?
+          if feature_enabled_and_claimant_present?
+            ClaimsApi::PoaAssignDependentClaimantJob.perform_async(
+              dependent_claimant_poa_assignment_service(poa_code:, file_number:, dependent_participant_id:,
+                                                        claimant_ssn:)
+            )
```

Comment:
how does this render in the sidekiq UI? wondering if we run the risk of showing the file_number, PID, SSN, etc;

Reply:
No PII, just the `DependentClaimantPoaAssignmentService` object ID:

![sidekiq](https://github.com/user-attachments/assets/22b04b99-e6fd-4268-ac15-2324e1c4ab59)